Support directives on directive definitions in GraphQL (#19171)
prettier/prettier: changelog_unreleased/graphql/19171.md

#### Support directives on directive definitions (#19171 by @YBJ0000)

> [Directives on directive definitions](https://github.com/graphql/graphql-js/pull/4521) (and `extend directive` extensions) are now formatted instead of throwing a syntax error.

Reviewer: ```suggestion
[Directives on directive definitions](https://github.com/graphql/graphql-js/pull/4521) (and `extend directive` extensions) are now supported.
```
